test/e2e/runner/main.go

```diff
@@ -189,17 +210,24 @@ func NewCLI() *CLI {
 	})
 
 	cli.root.AddCommand(&cobra.Command{
-		Use:   "logs",
-		Short: "Shows the testnet logs",
+		Use:     "logs [node (optional)]",
+		Short:   "Shows the testnet or a specefic node's logs",
+		Example: "runner logs valiator03",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 1 {
+				return execComposeVerbose(cli.testnet.Dir, "logs", args[0])
+			}
 			return execComposeVerbose(cli.testnet.Dir, "logs")
 		},
 	})
 
 	cli.root.AddCommand(&cobra.Command{
-		Use:   "tail",
+		Use:   "tail [node (optional)]",
```

Comment:
`[]` is always used to denote optional arguments, by convention, so adding (optional) is redundant: https://en.wikipedia.org/wiki/Usage_message#Pattern

Reply:
Thanks good to know